### PR TITLE
Start the scheduler same as worker in conda env

### PR DESCRIPTION
**Pull request by aktech**
Quoting myself from an internal comment:

The dask distributed checks the versions of some packages, which should match on scheduler (and worker) and client.
These packages should match on scheduler:
https://github.com/dask/distributed/blob/44b2358e33a0738c4c70ca96db4242636245e07d/distributed/versions.py#L28-L31

And we don't have it in the dask-worker image (which is used by scheduler as well):
https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/image/dask-worker/environment.yaml

### CODE REVIEW

**Comment by costrouc** [comment]
Will we run into version inconsistencies when the jupyterlab instance has a different version of blosc and l4z? I think we need to launch the dask scheduler similar to https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/image/dask-worker/postBuild#L6-L11. If we look into dask-gateway https://github.com/dask/dask-gateway/blob/0.6.1/dask-gateway-server/dask_gateway_server/managers/base.py#L182-L191 we see how the scheduler in run. So we'll need to patch in the same way that the dask-worker was to ensure that the environment is consistent.

**Comment by costrouc** [comment]
This is how we tell dask-gateway to launch with that script https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/infrastructure/dask-gateway.yaml#L16-L19

**Comment by costrouc** [comment]
Yeah this isn't ideal but dask-gateway as far as I can tell doesn't give us any other options. We also need to make sure that conda-store pvc is mounted in the scheduler so that it can run the selected environment

**Comment by aktech** [comment]
@costrouc I think that makes sense, thanks for reviewing. I will fix this.

**Comment by aktech** [comment]
I have added the command to start scheduler to same as worker and mounted the conda-store to scheduler here: https://github.com/Quansight/qhub-terraform-modules/pull/7

File: qhub_ops/template/{{ cookiecutter.repo_directory }}/infrastructure/dask-gateway.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ gateway:
       def worker_profile(options):
            config = profiles[options.profile]
            config['worker_cmd'] = '/opt/conda-run'
+           config['scheduler_cmd'] = '/opt/conda-run'
```
Comment (costrouc) [inline qhub_ops/template/{{ cookiecutter.repo_directory }}/infrastructure/dask-gateway.yaml]: We need to rename the `conda-run` script to something like `conda-run-worker` and create another script `conda-run-scheduler` reason being is that at the moment `conda-run` only runs the dask-gateway-worker see https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/image/dask-worker/postBuild#L10.

Comment (aktech) [reply qhub_ops/template/{{ cookiecutter.repo_directory }}/infrastructure/dask-gateway.yaml]: Done, thanks!

**Review by costrouc — APPROVED** [review]
Looks good to me

**Pull request merged**